library/item/item_setting/build.gradle

```diff
@@ -12,6 +12,15 @@ qslModule {
 		core {
 			api("qsl_base")
 			testmodOnly("resource_loader")
+			testmodOnly("lifecycle_events")
+			testmodOnly("networking")
+		}
+		data {
+			testmodOnly("registry_entry_attachments")
```

Comment:
Is this actually used in the testmod? or are transitive APIs not pulled with ICR

Reply:
I added REA when I added all of its impl dependencies, it didn't actually need to be pulled in